src/plugin-host/plugin-events/event-arguments/no-arguments-plugin-event-arguments.ts

```diff
@@ -0,0 +1,17 @@
+import { PluginEventArguments } from "../plugin-event-arguments";
+import { Chat } from "../../../chat/chat";
+
+/**
+ * Leaderboard Reset Plugin Event Arguments.
+ * Contains chat that has been reset.
+ */
+export class NoArgumentsPluginEventArguments extends PluginEventArguments
```

Comment:
What is this event for? It's not clear from the name and the lack of correct comments what it means without looking up where it's used.

Reply:
See docs